README.md

```diff
@@ -42,22 +42,30 @@ If the steps worked succesfully, you will see pycurl's version includes "OpenSSL
 PycURL/7.43.0.6 libcurl/7.74.0 (SecureTransport) OpenSSL/1.1.1i zlib/1.2.11 brotli/1.0.9 zstd/1.4.8 libidn2/2.3.0 libssh2/1.9.0 nghttp2/1.42.0 librtmp/2.3
 ```
 
-##  To run the application
-
-First, run `scripts/bootstrap.sh` to install dependencies.
+### `environment.sh`
 
-Create a local environment.sh file containing the following:
+Create and edit an environment.sh file.
 
 ```sh
 echo "
 export NOTIFICATION_QUEUE_PREFIX="YOUR_OWN_PREFIX"
 export NOTIFY_ENVIRONMENT="development"
-export FTP_HOST="YOUR_IP_ADDRESS"
+export FTP_HOST="localhost"
 export FTP_USERNAME="YOUR_LDAP_USERNAME"
 export FTP_PASSWORD="YOUR_LDAP_PASSWORD"
 "> environment.sh
 ```
 
+Things to change:
+
+- Replace `YOUR_OWN_PREFIX` with `local_dev_<first name>`.
+- Replace `YOUR_LDAP_USERNAME` with the one you use to sign in to your machine.
+- Replace `YOUR_LDAP_PASSWORD` with the one you use to sign in to your machine (yes really!).
```

Comment:
In a subsequent PR, we may wish to change this so that we aren't encouraging people to store their LDAP password unecrypted in a file on their laptop. If we have to provide it, we should provide it something like

```
 FTP_PASSWORD=passwordhere make run-celery
```

but that change can be made separately to this PR

Reply:
I agree, it surprised me this was necessary. I'm afraid your suggestion (adding a space before the command) doesn't work for me for some reason. For now, I've added a final commit to indicate it's not ideal, since:

- I'm not sure how easy it will be to properly avoid doing this.
- It's unclear how often people actually need to spin up this repo.

I think having a warning is better than letting people assume it's OK.

I'll leave this PR up for another day in case you have any comments on the change.